{{cookiecutter.repo_name}}/docs/source/conf.py

```diff
@@ -56,6 +56,12 @@
 ]
 
 autosummary_generate = True
+# This skips generating an autodoc of the test module
+# when using the autosummary directive that is included
+# by default in api.rst
+autodoc_mock_imports = [
+    '{{cookiecutter.project_name}}.tests'
```

Comment:
```suggestion
    '{{cookiecutter.package_name}}.tests'
```

The project name can have any characters and spaces -- we should use the package_name here.